src/Mundo.java

```diff
@@ -69,6 +69,68 @@ public void reiniciaMundo()
                 ,{2,0,0,2,2,2,2,20,20,20,20,20,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,0,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
                 ,{5,5,5,5,5,5,5,20,20,20,20,20,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
                 ,{0,0,0,0,0,0,0,20,20,20,20,20,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}};
+
+        mundoQ4 = new int[][] // 60x60 quarto quadrante / 40 semaforos / 17 cruzamentos
+                {{1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,0,8,0,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,0,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,0,8,0,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,0,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,8,2,7,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}
+                ,{1,1,1,1,1,1,1,1,1,70,70,70,70,70,0,0,0,0,0,0,71,71,71,71,71,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,72,72,72,72,72,0,0,0,0,0,0,0,73,73,73,73,73,0,0,0}
+                ,{0,0,0,69,69,69,69,69,1,70,70,70,70,70,6,6,6,6,6,6,71,71,71,71,71,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,72,72,72,72,72,6,6,6,6,6,6,6,73,73,73,73,73,6,6,6}
```

Comment:
E esse. A intersecção 69 começa nessa linha, mas a 70 na linha acima.

Reply:
essa 69 foi adicionada pela quebra de um cruzamento mas se quiser eu do uma alinhada

Reply:
É melhor, pq na linha 84, a intersecção 70 encosta na 69 sem passar por uma aresta

Reply:
a n pera se ta perguntando se esta correto, se for isso está correto sim ela é uma linha abaixo e uma linha a esquerda